plugins/woocommerce/src/Internal/Admin/WcPayWelcomePage.php

```diff
@@ -189,6 +190,22 @@ public function allowed_promo_notes( $promo_notes = [] ): array {
 		return $promo_notes;
 	}
 
+	/**
+	 * Adds the WooPayments incentive badge to the onboarding task.
+	 *
+	 * @param string $badge Current badge.
+	 *
+	 * @return string
+	 */
+	public function onboarding_task_badge( string $badge ): string {
+		// Return early if the incentive must not be visible.
+		if ( ! $this->must_be_visible() ) {
+			return $badge;
+		}
+
+		return $this->get_incentive()['task_badge'] ?? $badge;
```

Comment:
I would probably go extra-defensive here and check is `$this->get_incentive()` is not empty and only then try to get value of `$this->get_incentive()['task_badge']`

Reply:
In this case the _null coalescing operator_ give us safe access to any nested value on an empty array. That's the beauty of it, and why I love shorthands 😄